### PR TITLE
Update setup_env to match the new dialogue application

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

File: setup_env/applications.yaml [hunk 1]
```diff
@@ -42,10 +66,6 @@ dialogue:
     send_to:
       default:
         transport_name: "invalid broken transport"
-    vxpolls:
-      prefix: vxpolls
-    middleware:
-      - logging_mw: vumi.middleware.logging.LoggingMiddleware
```
Comment (hodgestar) [inline setup_env/applications.yaml:48]: Maybe we should keep the logging middleware?

**Comment by hodgestar** [comment]
Other than the suggestion to keep the logging middleware, :+1:.

**Comment by justinvdm** [comment]
Ready for re-review

**Comment by hodgestar** [comment]
:+1:

File: setup_env/applications.yaml [hunk 2]
```diff
@@ -32,8 +32,32 @@ bulk_message:
 dialogue:
   class: go.apps.dialogue.vumi_app.DialogueApplication
   config:
-    batch_size: 2
     metrics_prefix: dialogue_application
+    env:
+      NODE_PATH: "/usr/lib/node_modules"
```
Comment (jerith) [inline setup_env/applications.yaml:37]: I'm pretty sure this isn't cross-platform.

Comment (justinvdm) [inline setup_env/applications.yaml:37]: Agreed. We shouldn't be globally installing and using node modules anyways. Hopefully we can solve this along with the rest of the js environment set up stuff in one fell swoop when we get around to it.

**Pull request merged**